styles/style.css

```diff
@@ -0,0 +1,237 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700');
+
+* {
+  box-sizing: border-box;
+  
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+span {
+  display: block;
+  text-align: center;
+}
+
+h2 {
+  text-align: center;
+  padding: 40px;
+}
+
+body {
+  background-color: #ffffff;
+  color: #434344;
+  font-family: "Montserrat", sans-serif;
+  font-weight: 400;
+}
+
+section {
+  border-bottom: solid 2px black;
+}
+
+section:last-of-type {
+  border-bottom: none;
+}
+
+.container {
+  max-width: auto;
+  display: grid;
```

Comment:
If you want to make sure your webpage more responsive to changes, you could try removing the grid display from the container.  (I played with this, and removing grid display from your container causes some other unintended changes (that should be easily fixable), but be aware of that!)